src/main/php/PHPMD/Rule/Naming/LongVariable.php

```diff
@@ -109,7 +116,11 @@ protected function checkMaximumLength(AbstractNode $node)
     {
         $threshold = $this->getIntProperty('maximum');
         $variableName = $node->getImage();
-        $lengthWithoutDollarSign = Strings::lengthWithoutSuffixes($variableName, $this->getSubtractSuffixList()) - 1;
+        $lengthWithoutDollarSign = Strings::lengthWithoutPrefixesAndSuffixes(
+            \ltrim($variableName, '$'),
```

Comment:
Nit: I dont' think we normally name space functions
```suggestion
            ltrim($variableName, '$'),
```